actions/upload-repository/action.yml

```diff
@@ -45,27 +46,34 @@ runs:
         find build -type f | xargs ls -lh
       shell: bash
 
-    - uses: actions/jekyll-build-pages@44a6e6beabd48582f863aeeb6cb2151cc1716697 # 1.0.13
+    - name: Render repository state in HTML
+      uses: docker://pandoc/core:3.5
```

Comment:
Should we pin by hash here?

Reply:
yeah maybe that is better -- it does not look like a fast moving target anyway